internal/proxy/github_test.go

```diff
@@ -29,7 +29,11 @@ func TestGHWebHookRequest(t *testing.T) {
 	w := httptest.NewRecorder()
 	req := httptest.NewRequest("GET", "http://proxy", bytes.NewBuffer(gh))
 	req.Header.Add("User-Agent", "GitHub-Hookshot"+uuid.NewV4().String())
-	p.handleGitHubRequest(w, req, proxyLogger)
+	ns, okToForward := p.handleGitHubRequest(w, req, proxyLogger)
+
+	assert.Equal(t, ns, "namespace-jenkins")
+	assert.False(t, okToForward, "It should not be ok to forward")
```

Comment:
You are testing the "good path" can you handle the "bad path" too ?

Reply:
sure, I will think of some bad paths and add them